src/index.html

```diff
@@ -22,6 +22,359 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link header__nav-link--is-active"
+              data-qa="card"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="cards">
+      <div class="cards__item">
+        <img
+          class="cards__photo"
+          src="images/imac.jpeg"
+          alt="iMacA1419"
+        />
+        <div class="cards__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="cards__subtitle">Product code: 195434</div>
+
+        <div class="cards__rating">
+          <ul class="stars stars--count-4">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+          <p class="cards__review-count">Reviews: 5</p>
+        </div>
+        <div class="cards__price">
+          <span class="cards__price-label">Price:</span>
+          <span class="cards__price-value">$2,199</span>
+        </div>
```

Comment:
```suggestion
        <p class="cards__price">
          <span class="cards__price-label">Price:</span>
          <span class="cards__price-value">$2,199</span>
        </p>
```
Consider leveraging semantic tags for elements